### DISCUSSION TITLE
How much time take for github sponsers ?

### DISCUSSION THREAD

**Discussion by Harries** [community/community#136342] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I submit GitHub Sponsors at 2024/08/05.but so far it is a pending status? the tips is that
```
 Your profile is pending approval, and isn't live yet.
Publish your GitHub Sponsors profile
You've submitted your profile for approval! Your profile will go live as soon as it's approved by a staff member.
```
Does anyone know why the review is so slow?Up to now, 15 days have passed

**Comment by srijon57** [comment, marked as answer]
Hello, my friend. Sometimes GitHub takes a lot of time.
The review process for GitHub Sponsors can vary in duration due to several factors, including the volume of applications and the availability of staff members to review them. Here are a few points to consider:

Volume of Applications: If there are a lot of applications being submitted around the same time, it can lead to longer review times.

Staff Availability: The availability of GitHub staff to review applications can also impact the timeline. If staff members are busy or if there are fewer reviewers available, it can cause delays.

Profile Details: The completeness and accuracy of the information provided in your profile can affect the review time. If there are any issues or missing details, it might take longer for the review to be completed.

Communication: Sometimes, delays can occur due to communication issues. It might be helpful to check your email or GitHub notifications to see if there are any messages from the GitHub team requesting additional information or clarification.

If it has been 15 days and your profile is still pending, you might want to consider the following steps:

Check Your Profile: Ensure that all the required information is accurately filled out and that there are no missing details.

Contact GitHub Support: Reach out to GitHub Support to inquire about the status of your application. They might be able to provide more information or expedite the review process.

Community Feedback: Sometimes, other users in the GitHub community might have experienced similar delays and could offer insights or advice based on their own experiences.

**Comment by Harries** [comment]
it‘s be done
